src/IW/Core/PGArray'.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE DeriveAnyClass #-}
+
+module IW.Core.PGArray'
+    ( PGArray' (..)
+    ) where
+
+import Database.PostgreSQL.Simple.Types (PGArray (..))
+import Database.PostgreSQL.Simple.FromField (FieldParser, pgArrayFieldParser)
+import Database.PostgreSQL.Simple.ToField (Action (..))
+
+
+newtype PGArray' a = PGArray' { fromPGArray' :: [a] }
+    deriving stock (Generic, Show, Eq)
+    deriving anyclass (ToJSON)
```

Comment:
`Eq` instance for `newtype`s should always be derived using `newtype` deriving strategy.

`ToJSON` should be derived using `newtype` strategy in this particular use case.

Reply:
Got it. Will fix.